src/brouwers/shop/payments/sisow/service.py

```diff
@@ -0,0 +1,91 @@
+from functools import lru_cache
+from typing import Iterator, List
+from urllib.parse import unquote
+
+from django.urls import reverse
+
+from ...models import Payment, ShopConfiguration
+from .api import NS, calculate_ideal_sha1, calculate_sha1, xml_request
+from .constants import Payments
+from .exceptions import InvalidIssuerURL
+
+
+class iDealBank:
+
+    def __init__(self, _id: str, name: str):
```

Comment:
Is this some kind of new python syntax where you can declare types of the args? Haven't seen it before.

Reply:
yeah, [type hinting](https://docs.python.org/3.6/library/typing.html) is new since 3.5

It's static, not at runtime, but makes the intent of the code a lot more obvious.